test/sav/run.c

```diff
@@ -53,7 +53,6 @@ static int run_autopilot(const char *input_saved_game, const char *output_saved_
     printf("Running autopilot: %s --> %s in %d ticks\n", input_saved_game, output_saved_game, ticks_to_run);
     signal(SIGSEGV, handler);
 
-    chdir("data/sav");
```

Comment:
Changing dir is no longer necessary?

Reply:
No, when I added a check to fix the warning all the tests reported as failed as that dir does no longer exist. I'm guessing that's leftover code from a time when tests were structured differently.